util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -65,12 +76,24 @@ public StatChecker(LogWrapper logCtx, Set<String> samplePlaces, boolean verbose)
   // statVarObservation nodes and save it into seriesSummaryMap.
   public synchronized void extractSeriesInfoFromGraph(McfGraph graph) {
     for (Map.Entry<String, McfGraph.PropertyValues> node : graph.getNodesMap().entrySet()) {
-      if (shouldExtractSeriesInfo(node.getValue())) {
+      if (isSvObWithNumberValue(node.getValue()) && shouldExtractSeriesInfo(node.getValue())) {
         extractSeriesInfoFromNode(node.getValue());
       }
     }
   }
 
+  // Given a graph, for each node that is a statVarObservation node with a number value, check for
+  // any value inconsistencies. Return false if there are any svObsValueInconsistencies found.
+  public synchronized boolean checkSvObsInGraph(McfGraph graph) {
```

Comment:
I previously thought you were going to merge this into extractSeriesInfoFromGraph() and have a single processGraph API... but since we have two, its simple enough to drop the synchronized, and make svObValues a concurrentHashMap?

Reply:
done